### ISSUE TITLE
Inline buttons/badges

### ISSUE THREAD

**Issue by chrisjsewell** [executablebooks/sphinx-panels#20]
Currently, the `link-button` directive is a block element, which creates a reference node wrapped in a paragraph node. It would be nice though if you could also have an inline button, particularly for having multiple buttons in a row.
The main issue here is that using roles does not allow for options.
Maybe an option/logic could be added so that the reference is not wrapped, but instead added as a child to the preceding "sibling" element. Although this would only allow for buttons at the end of paragraphs (but you could chain them).

Note also that docutils requires that the reference node is wrapped in a `TextElement` type node: https://github.com/docutils-mirror/docutils/blob/e88c5fb08d5cdfa8b4ac1020dd6f7177778d5990/docutils/writers/html4css1/__init__.py#L1419

**Comment by chrisjsewell** [comment]
Note there are also the bootstrap badge classes: https://getbootstrap.com/docs/4.4/components/badge/#links

**Comment by chrisjsewell** [comment]
Perhaps also a `button-group`, a bit like what is used in pandas:

```html
<div class="d-flex flex-row">
  <span class="badge gs-badge-link">
    <p><a class="reference internal" href="intro_tutorials/01_table_oriented.html#min-tut-01-tableoriented"><span class="std std-ref">To introduction tutorial</span></a></p>
  </span>
  <span class="badge gs-badge-link">
    <p><a class="reference internal" href="dsintro.html#dsintro"><span class="std std-ref">To user guide</span></a></p>
  </span>
</div>
```

**Comment by chrisjsewell** [comment]
Perhaps something like `` :link-badge:`some text <link> {class1 class2}` ``

**Comment by chrisjsewell** [comment]
closed by #27
